src/Symfony/Component/Intl/AbstractStaticBundle.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Intl;
+
+use Symfony\Component\Intl\Data\Bundle\Reader\BufferedBundleReader;
+use Symfony\Component\Intl\Data\Bundle\Reader\BundleEntryReader;
+use Symfony\Component\Intl\Data\Bundle\Reader\BundleEntryReaderInterface;
+use Symfony\Component\Intl\Data\Bundle\Reader\JsonBundleReader;
+
+/**
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+abstract class AbstractStaticBundle
```

Comment:
```suggestion
abstract class ResourceBundle
```

Not a fan of the current name. I'd stick to the terminology we already have. I think neither "Abstract" nor "Static" provide any valuable information about this class.